<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="TYSS-Sanjay" failures="1" tests="3" name="Test" time="33.33" errors="0" timestamp="22 May 2024 09:27:26 GMT">
  <testcase classname="com.comcast.crm.organizationtest.CreateOrganizationTest" name="createOrg" time="7.066"/>
  <testcase classname="com.comcast.crm.organizationtest.CreateOrganizationTest" name="createOrgWithIndustry" time="7.385"/>
  <testcase classname="com.comcast.crm.organizationtest.CreateOrganizationTest" name="createOrganizationWithphonenumbertest" time="7.312">
    <failure type="java.lang.AssertionError" message="expected [Whatsapp_523] but found [9876766400]">
      <![CDATA[java.lang.AssertionError: expected [Whatsapp_523] but found [9876766400]
at com.comcast.crm.organizationtest.CreateOrganizationTest.createOrganizationWithphonenumbertest(CreateOrganizationTest.java:81)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- createOrganizationWithphonenumbertest -->
</testsuite> <!-- Test -->
